### DISCUSSION TITLE
How to clear cached moves before starting new `go depth n`

### DISCUSSION THREAD

**Discussion by fishchess** [official-stockfish/Stockfish#4275] [Q&A] [answered]
when I run `go depth 10 ` 

`info depth 10 seldepth 13 multipv 1 score cp 25 nodes 20978 nps 1165444 hashfull 10 tbhits 0 time 18 pv e2e4 c7c5 g1f3 b8c6 f1c4 e7e6 e1g1 g8f6`

when I run `go depth 10 ` again 

`info depth 10 seldepth 14 multipv 1 score cp 33 nodes 8881 nps 740083 hashfull 3 tbhits 0 time 12 pv e2e4 c7c5 g1f3 b8c6 b1c3 g8f6 d2d4 c5d4 f3d4`

I see in second search used first cached search move

how to clear cached search before starting new search?

**Comment by vondele** [comment, marked as answer]
Help on how to clear caches is available on Discord channels and our wiki page
